source/transfer_list.rst

```diff
@@ -845,6 +845,39 @@ Entries related to Trusted Firmware
 The following entry types are defined for Trusted Firmware projects,
 including TF-A, OP-TEE and Hafnium:
 
+.. list-table::
```

Comment:
minor: Would be good to give this table a name. It looks a bit odd that this table appears between Tables 12 and 13 in the rendered doc.